src/tooling/piral-cli/src/injectors/piral.ts

```diff
@@ -32,11 +32,16 @@ export default class PiralInjector implements KrasInjector {
 
   setOptions() {}
 
-  sendResponse(path: string, target: string, dir: string, url: string): KrasResponse {
+  sendResponse(path: string, target: string, dir: string, url: string, recursionDepth: number = 0): KrasResponse {
+
+    if(recursionDepth > 10) {
```

Comment:
Please use prettier for your formatting (VS Code command palette "Format Document") -> space between `if` and the `(`.

Reply:
Maybe make the 10 a const on the top of the file. Also I think ~4 should be sufficient (or even 3).

Reply:
adapted; ran prettier, and created the constant `maxRetrySendResponse` with value of 4